mgmtsystem_extended_mrp/views/mgmtsystem_nonconformity_views.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- Copyright 2019 Marcelo Frare (Associazione PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+     Copyright 2019 Stefano Consolaro (Associazione PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl) -->
+     
+<odoo>
+
+    <!-- inherit NC form view 
+            adding workcenter -->
+    <record id="view_mgmtsystem_nonconformity_form" model="ir.ui.view">
+        <field name="name">view.mgmtsystem.nonconformity.department</field>
+        <field name="model">mgmtsystem.nonconformity</field>
+        <field name="inherit_id" ref="mgmtsystem_nonconformity.view_mgmtsystem_nonconformity_form"/>
+        <field name="arch" type="xml">
+            <field name="create_date" position="after">
+                <field name="workcenter_id"/>
+            </field>
+        </field>
+    </record>
```

Comment:
workcenters are not always used. It depends on the user's manufacturing configuration.
Basically, only if the user has this group: `group_mrp_routings` (Manage Work Order Operations)

IMO you should add this group to either the view or the field, so that it's only shown in this case.

Reply:
Done, I hope in the right way.